app/src/main/java/org/simple/clinic/summary/teleconsultation/messagebuilder/TeleconsultationSmsMessageBuilder.kt

```diff
@@ -0,0 +1,19 @@
+package org.simple.clinic.summary.teleconsultation.messagebuilder
+
+import org.simple.clinic.summary.PatientTeleconsultationInfo
+import javax.inject.Inject
+
+class TeleconsultationSmsMessageBuilder @Inject constructor() : TeleconsultationMessageBuilder {
+
+  // TODO (SM): 2020-08-24 Change the deep link
+  override fun message(patientTeleconsultationInfo: PatientTeleconsultationInfo): String {
+    val message = StringBuilder("Teleconsult request")
+        .appendLine()
+        .appendLine()
+        .appendLine("Review patient & log a record:")
```

Comment:
I wonder if we should add comments in this class and in the other message builder that we are explicitly not localizing these strings because we do not know the preferred language of the recipient. Otherwise, someone who is well meaning might end up extracting these to resources in the future without knowing.

Reply:
Yep, will do that in later commits 👍🏾

Reply:
NVM, ended up changing it in this PR.